tsconfig.json

```diff
@@ -24,6 +24,7 @@
   },
   "include": [
     "./@types/styles.d.ts",
+    "./@types/json.d.ts",
```

Comment:
maybe it'd be easier to just have a `custom.d.ts` file with everything inside of it (css, scss and json and any other future stuff)?

Reply:
Good point